dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -69,7 +74,8 @@ check_not_exporting_variables() {
         patterns+=(-e "export[[:blank:]]\+${variable}")
     done
 
-    git -C "${RIOTBASE}" grep "${patterns[@]}"
+    git -C "${RIOTBASE}" grep "${patterns[@]}" \
+        | sed '1i \\nVariables must not be exported:'
```

Comment:
```suggestion
        | sed -e '1i \\nVariables must not be exported:' -e 's/^/\t/'
```
That will indent the following lines.

Reply:
I will try it on mac too.